.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: Pipeline
+on:
+  pull_request:
+    branches:
+      - master
+      - v*
+      - litmus-v2
+
+jobs:
+  backend-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          cd litmus-portal
+          make backend-services-checks
+
+  frontend-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          cd litmus-portal
+          make frontend-services-checks
+
+  docker-build-graphql-server:
+    runs-on: ubuntu-latest
+    needs:
+      - backend-checks
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
```

Comment:
```
actions/checkout@v2
```
Can you also check this at other places?

Reply:
Done